bindings/node/lib/mongocryptdManager.js

```diff
@@ -22,7 +24,7 @@ class MongocryptdManager {
     } else {
       // TODO: eventually support connecting on Linux Socket for non-windows,
       // blocked by SERVER-41029
```

Comment:
Might as well clear this up, it was closed as a won't fix

Reply:
I don't have context on this ticket.  Do you want the logic removed or just the comment?